### ISSUE TITLE
Results on the dataset

### ISSUE THREAD

**Issue by yarqian** [VivianSZF/Stereo-Waterdrop-Removal#1]
Many thanks for your work. I test the pretrained model on the dataset and get the same results with that on paper. But when I trained the model using the default settings, the PSNR is only 25.487 (26.064 in paper). So I am sorry to ask do you have any suggestions about this? Thank you.

**Comment by VivianSZF** [comment]
There are several factors I can think of:
1. The batch size is important. We train our model with batch size 3. BS that is smaller than 3 will lead to a drop in the performance.
2. Adjust the learning rate. (e.g., #steps to decay the lr)
3. Different versions of pytorch, CUDA, GPU may affect the results a little bit. The average performance with the versions in Readme and Nvidia TITAN RTX should be around 26.

**Comment by yarqian** [comment]
For the batch size and the learning rate, I used the settings in your codes. For the version of  pytorch, I directly install pytorch by using the requirements.txt. For the GPU, I also used TITAN RTX.

**Comment by VivianSZF** [comment]
I see. Here are some other suggestions:
1. Avoid the overfitting. From my experience, you need to stop the training at around 60-80 epochs(It depends as there is some randomness of the training, e.g. the initialization).
2. Adjust the learning rate. The default one might not be the best one all the time.

**Comment by yarqian** [comment]
ok, maybe you should fix the random seed as first. Anyway thank you for your suggestions and I will try it later.

**Comment by yarqian** [comment]
I think the key problem in your code is that the performance is highly affected by the initialization. I trained the model two times with the same settings, but the performance varies greatly.

**Comment by VivianSZF** [comment]
Thx. I'll fix the random seed later.
